packages/ejson/has_size.js

```diff
@@ -0,0 +1,29 @@
+// Like `_.size(obj) === n` but faster by looking at at most `n+1`
+// items
+hasSize = function _hasSize(obj, n) {
```

Comment:
do we still need this `x = function _x` naming pattern with the newest ecmascript package? @benjamn?